modules/twigextensionmodule/twigextension/TwigExtension.php

```diff
@@ -0,0 +1,39 @@
+<?php
```

Comment:
Use spaces for indenting please (yeah I know. yuk.) we only use spaces in PHP bc of psr complience.